app/views/govuk_publishing_components/components/docs/govspeak.yml

```diff
@@ -911,3 +912,28 @@ examples:
             <p>Deforested area. Credit: Blue Ventures-Garth Cripps</p>
           </figcaption>
         </figure>
+  different_font_sizes:
```

Comment:
Suggestion: `large_font_sizes`

Reply:
I guess assuming we'll never want "small" govspeak?

If we know we're only ever going to support one other size, maybe a boolean parameter would be better than a string.

i.e.

```
large_font_size: true
# instead of
font_size: l
```

?

Reply:
Exactly, yes.